storage-s3-dynamodb/src/test/scala/io/delta/storage/ExternalLogStoreSuite.scala

```diff
@@ -219,6 +221,60 @@ class ExternalLogStoreSuite extends org.apache.spark.sql.delta.PublicLogStoreSui
   protected def shouldUseRenameToWriteCheckpoint: Boolean = false
 }
 
+///////////////////////////////////
+// S3DynamoDBLogStore Test Suite //
+///////////////////////////////////
+
+class S3DynamoDBLogStoreSuite extends AnyFunSuite {
+  test("getParam") {
+    import S3DynamoDBLogStore._
+
+    val sparkPrefixKey = "spark.io.delta.storage.S3DynamoDBLogStore.ddb.tableName"
+    val basePrefixKey = "io.delta.storage.S3DynamoDBLogStore.ddb.tableName"
+
+    // Sanity check
+    assert(sparkPrefixKey == SPARK_CONF_PREFIX + "." + DDB_CLIENT_TABLE)
```

Comment:
nit: typically these should be require as this failure does not mean some thing is wrong in the production code... this failure means something is wrong in the test code.